.shellcheckrc

```diff
@@ -0,0 +1,2 @@
+# Allow opening any 'source'd file, even if not specified as input
+external-sources=true
```

Comment:
Now when this is enabled we could/should remove the `beakerlib` installation?

Reply:
We should probably remove it. I don't know why it doesn't work in GA. When I install beakerlib locally from rpm ShellCheck is happy. I will remove the cloning and install beakerlib from the workflow.

Reply:
Ack, let's remove it, as it seems it does not have any effect on the evaluation.